src/pages/blog/starlingx-debian-technology-preview-in-release-7.md

```diff
@@ -0,0 +1,376 @@
+        (All-in-one Install - Serial; Console)
+
+          menu32               = no default system install type ; requires manual select
+
+          disk                 = Disk Boot
+
+          standard>serial      = Controller Install - Serial Console
+
+          standard>graphical   = Controller Install - Graphical Console
+
+          aio>serial           = All-in-one Install - Serial Console
+
+          aio>graphical        = All-in-one Install - Graphical Console
+
+          aio-lowlat>serial    = All-in-one (lowlatency) Install - Serial Console
+
+          aio-lowlat>graphical = All-in-one (lowlatency) Install - Graphical Console
+
+      The ISO also contains the `pxeboot/samples/pxeboot_setup.sh`
+      script that can be used to automatically setup both the BIOS and
+      UEFI grub files for a specific install.
+
+          ./feed/pxeboot/samples/pxeboot_setup.sh --help
+
+          Usage: ./pxeboot_setup.sh [Arguments Options]
+
+          Arguments:
+
+          -i | --input   <input path>     : Path to pxeboot.cfg.debian and efi-pxeboot.cfg.debian grub template files
+          -o | --output  <output path>    : Path to created pxeboot.cfg.debian and efi-pxeboot.cfg.debian grub files
+          -p | --pxeboot <pxeboot path>   : Offset path between /pxeboot and bzImage/initrd
+          -f | --feed    <feed path>      : Offset path between http server base and mounted iso
+          -u | --url     <pxe server url> : The pxeboot server's URL
+
+          Options:
+
+          -h | --help                     : Print this help info
+          -b | --backup                   : Create backup of updated grub files as .named files
+          -d | --device <install device>  : Install device path ; default: /dev/sda
+          -s | --system <system install>  : System install type ; default: 3
+
+          0 = Disk Boot
+          1 = Controller Install - Serial Console
+          2 = Controller Install - Graphical Console
+          3 = All-in-one Install - Serial Console       (default)
+          4 = All-in-one Install - Graphical Console
+          5 = All-in-one (lowlatency) Install - Serial Console
+          6 = All-in-one (lowlatency) Install - Graphical Console
+
+          Example:
+
+          pxeboot_setup.sh -i /path/to/grub/template/dir
+                           -o /path/to/target/iso/mount
+                           -p pxeboot/offset/to/bzImage_initrd
+                           -f pxeboot/offset/to/target_feed
+                           -u http://###.###.###.###
+                           -d /dev/sde
+                           -s 5
+
+The remaining install steps are also completely unchanged. You can find them
+here:
+
+<https://docs.starlingx.io/deploy_install_guides/r6_release/bare_metal/aio_simplex_install_kubernetes.html>
+
+# Technology Preview Known Issues
+
+Known issues and workarounds with the StarlingX release 7.0 are the same as
+those for StarlingX release 7.0 based on CentOS.
```

Comment:
To end the post, you can use the general text we have at the end of the release overview blog posts to point people to the project resources, release notes, etc.:

For the complete list of updates and new features in StarlingX R7.0, check out the [release notes](https://docs.starlingx.io/releasenotes/r7-0-release-notes-bc72d0b961e7.html) and the [project documentation](https://docs.starlingx.io/).

Visit the StarlingX website today for further information about the project, check out the [code](https://opendev.org/starlingx), or download the [latest image](http://mirror.starlingx.cengn.ca/mirror/starlingx/release/) to try out the new features.